dojo/db_migrations/0201_alter_finding_options_finding_epss_percentile_and_more.py

```diff
@@ -0,0 +1,36 @@
+# Generated by Django 4.1.13 on 2024-02-11 15:32
+
+import django.core.validators
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
```

Comment:
Hi @quirinziessler,

could you update your PR against the latest dev? There is a db migrations 0201 and 0202 file already:
https://github.com/DefectDojo/django-DefectDojo/blob/dev/dojo/db_migrations/0202_alter_dojo_group_social_provider.py

Thus, you have to adapt this db migration file accordingly.

Reply:
Done 👍